src/modules/flow/process/output.c

```diff
@@ -76,7 +76,7 @@ output_write(struct output_data *output)
         } else if (r < 0) {
             if (errno == EINTR)
                 continue;
-            else if (errno == EAGAIN || errno == EWOULDBLOCK)
+            else if (errno == EAGAIN)
```

Comment:
Well, what are we going to do on those "non supported systems" if the values are different?